<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharWindow</class>
 <widget class="QWidget" name="CharWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>캐릭터 선택창</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>window_icon.png</normaloff>window_icon.png</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>캐릭터를 선택해주세요</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout_char">
    <item>
     <widget class="QPushButton" name="btn_warrior">
      <property name="text">
       <string>워리어</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_mase">
      <property name="text">
       <string>메이지</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_assasin">
      <property name="text">
       <string>어쌔신</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
